### DISCUSSION TITLE
Sharing data between databases

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#12826] [Q&A] [answered]
I'm trying to solve an issue where we want to share data from one database to another. Let's say we have one database containing a bunch of contacts and one database that holds the data for our website. We want the data to be separated but on our websites team page we want to show contacts from the contacts database. What options do I have?

**Comment by garayx** [comment]
You can open a new session to load the data from the contacts database,
[https://ravendb.net/docs/article-page/4.2/csharp/client-api/session/opening-a-session#syntax](https://ravendb.net/docs/article-page/4.2/csharp/client-api/session/opening-a-session#syntax)

**Reply by marcuslindblom** [reply]
That was not actually what I was looking for. It seems like RavenDB ETL is the thing I'm looking for.

**Comment by ayende** [comment, marked as answer]
You can use RavenDB ETL to handle that. That will allow you to create a channel in which you can push data from one database to another.
